tools/kicad/kicad_functions/src/vdiv.rs

```diff
@@ -96,22 +133,24 @@ fn calculate(config: &VoltageDividerConfig) -> Option<RRes> {
                             return None;
                         }
                     }
-                    panic_any(e) // No graceful way to handle this from the closure
+                    panic_any(e.to_string()) // No graceful way to handle this from the closure
                 }
-                _ => panic_any(e),
+                _ => panic_any(e.to_string()),
             },
         }
     })
 }
 
 /// `voltage_divider` computes values for resistor-based voltage dividers.
-/// - Usage: vdiv(<target voltage>, <divider expression>, <resistor series>, (min resistance), (max resistance))
-/// - Example: vdiv(5.1, "(R1+R2)/R2*0.8", "E96", 500e3, 700e3)
+/// - Usage: vdiv(<target voltage>, <divider expression>, <resistor series>,
+///               {(<min resistance>, <max resistance>)}, ({extra 1}, {extra 2}, ...))
+/// - Example: vdiv(5.1, "(R1+R2)/R2*E1", "E96", (500e3, 700e3), (0.8))
 /// - Output: (<closest voltage>, <R1 value>, <R2 value>, ...)
 /// There can be arbitrary many resistors in the divider, but they must be named "R1", "R2", etc.
 /// The computed optimal resistance values are also presented in this order. The minimal and maximal
-/// resistance limits are optional parameters, and only consider the sum of the resistances of all
-/// resistors defined in the expression.
+/// resistance pair is an optional parameter, and the limits only consider the sum of resistance of
+/// all resistors defined in the expression. The "extra" parameters are optional external inputs for
+/// the divider expression, and will be made available as "E1", "E2", etc. in order.
```

Comment:
For some reason this doc comment is not showing up in the documentation generated with `cargo doc`.

Reply:
The `vdiv` module is purposefully private (not `pub`), so to generate docs for it requires passing the `--document-private-items` flag to `cargo doc`. But thanks for noticing this, the formatting didn't like the unescaped `<>` so I've fixed that now in https://github.com/racklet/electronics-prototyping/pull/17/commits/7be3cd008297b83add39055224d2eacdf124c843.